src/Assetic/Extension/Twig/AsseticFilterFunction.php

```diff
@@ -11,19 +11,14 @@
 
 namespace Assetic\Extension\Twig;
 
-class AsseticFilterFunction extends \Twig_Function
+class AsseticFilterFunction extends \Twig_SimpleFunction
```

Comment:
Note that in Assetic 2.0, we would be able to stop using a custom class entirely. but I need to keep it like that for now for BC reasons.
I have some ideas in mind to refactor the Twig integration layer in Assetic to make the bundle cleaner (no overwrite of everything to handle the Symfony integration) but I will keep it for a separate PR.
Preparing the code for removal of this custom class will also be part of that delayed work.